opencxl/bin/cli.py

```diff
@@ -131,7 +133,7 @@ def start(
             threads.append(t_host)
             t_host.start()
         elif "host-group" in comp:
-            t_hgroup = threading.Thread(target=start_host_group, args=(ctx, config_file))
+            t_hgroup = threading.Thread(target=start_host_group, args=(ctx, config_file, no_hm))
             threads.append(t_hgroup)
             t_hgroup.start()
```

Comment:
define `hm_mode = not no_hm` here. We may change this in the future by combining with other options.